openff/toolkit/utils/_viz.py

```diff
@@ -92,15 +93,54 @@ class TopologyNGLViewStructure(Structure):
     def __init__(
         self,
         topology: Topology,
-        ext: str = "PDB",
     ):
         self.topology = topology
-        self.ext = ext.lower()
+        self.ext = "pdb"
         self.params: dict = dict()
         self.id = str(uuid.uuid4())
 
     def get_structure_string(self):
         with StringIO() as f:
-            self.topology.to_file(f, file_format=self.ext)
-            structure_string = f.getvalue()
+            if RDKIT_AVAILABLE:
+                from rdkit.Chem.rdmolfiles import PDBWriter  # type: ignore
```

Comment:
(blocking) Direct cheminformatics toolkit imports outside of toolkit wrappers have caused all sorts of havoc before. Even with the RDKIT_AVAILABLE guard I'm worried about this. Can this be refactored to put the RDKit import and associated code in RDKitToolkitWrapper?